<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationResultsDialog</class>
 <widget class="QDialog" name="CorrelationResultsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Correlation Results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Correlation test results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="testType">
     <property name="text">
      <string>Type of test : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pValue">
     <property name="text">
      <string>p-value : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statValue">
     <property name="text">
      <string>Statistical value : </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="testAgain">
       <property name="text">
        <string>Test again</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Send results to report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
